examples/tabular_model.py

```diff
@@ -0,0 +1,42 @@
+from neptune import new as neptune
+from neptune_fastai.impl import NeptuneCallback
+
+from fastai.basics import *
```

Comment:
I'm not familiar with data scientist practices and libraries, but I'm quite aware that it's a bad idea.

We're using `import *` in one place of our code (importing of our `neptune-<integration>` libraries), but we consciously define `__all__` in our integrations.

Reply:
I just copy-paste official examples from fastai :wink:. I 100% agree that it might be confusing.